alerts/google-quotas/metadata.yaml

```diff
@@ -26,6 +26,6 @@ alert_policy_templates:
   version: 1
 -
   id: quota-usage-mql
-  display_name: High quota usage
-  description: "Monitoring quota usage and will notify you if the usage rises above alert trigger threshold."
+  display_name: Quota usage
```

Comment:
we're deprecating the display_name field in the metadata.yaml file, since the UI only shows the policy's displayName field. could you please delete all display_name fields in this file as they're not used anywhere? sorry for the inconvenience!

Reply:
done.